package.json

```diff
@@ -89,17 +89,18 @@
   "dependencies": {
     "@paulcbetts/system-idle-time": "^1.0.4",
     "appdirectory": "^0.1.0",
+    "archiver": "^2.0.0",
     "async.map": "^0.5.2",
     "async.mapseries": "^0.5.2",
     "auto-launch": "^5.0.1",
     "electron-dl": "^1.9.0",
     "electron-spellchecker": "^1.2.0",
     "electron-squirrel-startup": "^1.0.0",
+    "extract-zip": "^1.6.5",
     "filesize": "^3.5.10",
     "keymirror": "0.1.1",
-    "winreg": "^1.2.3",
-    "archiver": "latest",
-    "extract-zip": "latest"
+    "node-forge": "^0.7.1",
```

Comment:
forge lib already exists in sfe-client code... why re-add here?  will end up having two copies of forge!  should consider re-using that existing lib.